deploy/operator/ztp/assisted-installer-crds-playbook.yaml

```diff
@@ -21,6 +21,10 @@
     - cluster_host_prefix: "{{ lookup('env', 'CLUSTER_HOST_PREFIX') }}"
     - external_subnet: "{{ lookup('env', 'EXTERNAL_SUBNET') }}"
     - service_subnet: "{{ lookup('env', 'SERVICE_SUBNET') }}"
+    - cluster_subnet_addn: "{{ lookup('env', 'CLUSTER_SUBNET_ADDN') }}"
```

Comment:
What is "addn"?

Reply:
I guess it means "additional"?

Reply:
Correct. It's a convention already used around (e.g. https://github.com/openshift-metal3/dev-scripts/blob/79aceff429a546b2559ee582b85a30f184389ef5/common.sh#L203) but I'm happy to use a different name if you have a better suggestion :)

Reply:
Since googling it yields no result, and it looks very similar to "ADDR", I think it's best to just write _ADDITIONAL

Reply:
Sure, done